libgap.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: @PACKAGE_NAME@
+Description: Library containing the kernel of GAP, a computer algebra system
+Version: @PACKAGE_VERSION@
+Requires: gmp
```

Comment:
Looking at the pkg-config manpage, shouldn't this actually be
```suggestion
Requires.private: gmp
```

Reply:
yes, I suppose user code won't normally touch GMP's API